core/nwb.base.yaml

```diff
@@ -37,6 +37,21 @@ datasets:
     doc: Description of the image.
     required: false
 
+- neurodata_type_def: ImageStack
+  neurodata_type_inc: VectorData
+  doc: A 1-dimensional ordered collection of images (by reference). The images should be stored in an Images type.
+  dims:
```

Comment:
I was thinking this would be a 3D column with shape (n_images, n_x_pixels, n_y_pixels).

Reply:
That would work as well. But that adds the constraint that all images must be the same size, which is usually, but not always the case. We would also have to add the option of a 4D dataset for non-grayscale images.

Reply:
On further thought, I agree. We should cover the situation that happens the vast majority of the time. We can always make a new type that is a 1D dataset of references to Image types that may be of different sizes when needed.